Automattic/jetpack
# Add Benefits Endpoint for Jetpack Disconnect Dialog

#### Changes proposed in this Pull Request:
The disconnect dialog project (see parBdM-fY-p2) needs an endpoint to return information about which benefits to highlight in the disconnect dialog. This PR adds a new endpoint at /site/benefits that calculates a list of benefits based on which features in Jetpack are currently being used. 

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
New feature, see parBdM-fY-p2 and https://github.com/Automattic/jetpack/pull/13118.

#### Testing instructions:

* Check out the branch
* Use Postman or something similar to make a request to https://:site/wp-json/jetpack/v4/site/benefits
* Verify that each of the items returned by the API is returned correctly.
* Verify that the data is only retrieved if the user is logged in and able to manage Jetpack.

#### Proposed changelog entry for your changes:
* Added an endpoint at wp-json/jetpack/v4/site/benefits for showing how Jetpack is being used on the site.
